### PR TITLE
F41/404

### PR DESCRIPTION
**Pull request by iruseg00**

### CODE REVIEW

File: app/src/components/404/404.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import style from "./style.module.scss";
+import { NavLink } from "react-router-dom";
+import nf404 from '../../assets/img/404.png';
+
+class NotFound extends React.Component {
```
Comment (i-barysiuk) [inline app/src/components/404/404.js]: no logic then use functional component

File: app/src/components/404/404.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  {
+    return(
+        <div className={style.container}>
+              <img src={nf404} alt="page not found" className={style.img}/>
+          <div className={style.text}>
+            Приносим свои извинения за предоставленные неудобаства, запрашиваемая вами страница не найдена или удалена.
+            Проверьте корректность вашего URL .
+          </div>
+          <div className={style.buttons}>
+            <NavLink className={style.button} onClick={this.props.history.goBack}>Назад     </NavLink>
```
Comment (i-barysiuk) [inline app/src/components/404/404.js]: Use common buttons from antd
its helps our project looks same in the differents parts and pages

File: app/src/components/404/style.module.scss [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+//buttons
+$but-width: 90px;
+$but-height: 30px;
+$but-radius:7px;
+$but-color: #1e3bba;
+$but-bg: #1e3bba;
+
+.container
+{
+    height: 90%;
```
Comment (i-barysiuk) [inline app/src/components/404/style.module.scss]: why only 90?

File: app/src/components/404/style.module.scss [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+    width: $text-width;
+    align-self: center;
+    text-align: justify;
+    font-family: $text-font;
+}
+
+.buttons
+{
+    align-self: center;
+    width:50%;
```
Comment (i-barysiuk) [inline app/src/components/404/style.module.scss]: ????

File: app/src/views/dashboard/Dashboard.jsx [hunk 5]
```diff
@@ -2,16 +2,19 @@ import Menu from "../../components/menu/Menu";
 import React from "react";
 import { Switch, Route } from "react-router-dom";
 import Request from "../../components/request/Request";
+import NotFound from '../../components/404/404';
```
Comment (i-barysiuk) [inline app/src/views/dashboard/Dashboard.jsx]: it is not component 
its should be in view directory

File: app/src/components/404/404.js (hunk 2 above)
Comment (i-barysiuk) [inline app/src/components/404/404.js]: WHY nav link and history.goBack on same element
its necessary

File: app/src/components/404/404.js [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+import nf404 from '../../assets/img/404.png';
+
+class NotFound extends React.Component {
+  render() 
+  {
+    return(
+        <div className={style.container}>
+              <img src={nf404} alt="page not found" className={style.img}/>
+          <div className={style.text}>
+            Приносим свои извинения за предоставленные неудобаства, запрашиваемая вами страница не найдена или удалена.
```
Comment (i-barysiuk) [inline app/src/components/404/404.js]: Может все же стоить сделать контент связанный с универом и более яркий 
возможно схожий с логин пейджем

**Review by i-barysiuk — CHANGES_REQUESTED** [review]

File: app/src/components/404/404.js (hunk 2 above)
Comment (iruseg00) [reply app/src/components/404/404.js]: First, I made the buttons from antd, then the links, and the links turned out better as for me ... Buttons are like buttons)

File: app/src/components/404/style.module.scss (hunk 3 above)
Comment (iruseg00) [reply app/src/components/404/style.module.scss]: on 100 looks like it is stretched

**Review by vol4k — APPROVED** [review]

**Pull request merged**